lms/services/application_instance_getter.py

```diff
@@ -70,6 +70,12 @@ def provisioning_enabled(self):
             provisioning = False
         return provisioning
 
+    def canvas_sections_enabled(self):
+        try:
+            return self._get_by_consumer_key().canvas_sections_enabled
+        except ConsumerKeyError:
+            return False
```

Comment:
I think the `ApplicationInstanceGetter` service is a bit silly and should just return the `ApplicationInstance` model itself to the calling code. But I don't want to get into refactoring it now